assets/js/components/ClusterDetails/ClusterDetails.stories.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+
+import {
+  clusterFactory,
+  checksExecutionCompletedFactory,
+  hostFactory,
+} from '@lib/test-utils/factories';
+
+import ClusterDetails from './ClusterDetails';
+
+export default {
+  title: 'ClusterDetails',
+  components: ClusterDetails,
+  decorators: [
+    (Story) => (
+      <MemoryRouter>
+        <Story />
+      </MemoryRouter>
+    ),
+  ],
+};
+
+function ContainerWrapper({ children }) {
+  return (
+    <div className="max-w-7xl mx-auto px-4 sm:px-6 md:px-8">{children}</div>
+  );
+}
+
+export function Hana() {
```

Comment:
In the latest versions of storybook the way of writing stories has changed a bit https://storybook.js.org/docs/react/writing-stories/introduction

What do you think of having new stories written that way?
```js
export const Hana = {
  args: {
    clusterID: ...,
    clusterName: ...,
  },
  render: () => <ContainerWrapper><ClusterDetails ... /</ContainerWrapper>
}
```
[Accordion](https://github.com/trento-project/web/blob/main/assets/js/components/Accordion/Accordion.stories.jsx) and [CheckResultDetail](https://github.com/trento-project/web/blob/main/assets/js/components/ExecutionResults/CheckResultDetail/CheckResultDetail.stories.jsx) try to follow that way.

No strong opinions.

Reply:
Hey @nelsonkopliku ,
Updated how story book story was written.
Indeed, i prefer this way, as it lets you customize the view super easily from storbook's web. Pretty cool!